### ISSUE TITLE
Release or put actual SNAPSHOT to Sonotype OSS Nexus

### ISSUE THREAD

**Issue by foal** [Moocar/logback-gelf#66]
Please, release the project or put the actual SNAPSHOT to Sonotype OSS Nexus.

Now the overview describe the Static Fields but it does not work for 0.2 version.

**Comment by bitkid** [comment]
yes please .. the Field class is not present in in the jar file so static fields won't work.
